src/main/java/seedu/address/logic/commands/ResetCourseCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_COURSES;
+
+import seedu.address.model.Model;
+
+/**
+ * Resets the current listed students to the original state.
```

Comment:
There's a typo here, should be "courses" instead of students". Everything else LGTM

Reply:
Noted. Will amend it now.